src/snowcli/cli/snowpark/jobs/manager.py

```diff
@@ -3,29 +3,34 @@
 from pathlib import Path
 
 from snowcli.cli.common.sql_execution import SqlExecutionMixin
+from snowcli.output.printing import OutputData
 
 
 class JobManager(SqlExecutionMixin):
-    def create(self, compute_pool: str, spec_path: Path, stage: str):
+    def create(self, compute_pool: str, spec_path: Path, stage: str) -> OutputData:
         spec_filename = os.path.basename(spec_path)
         file_hash = hashlib.md5(open(spec_path, "rb").read()).hexdigest()
         stage_dir = os.path.join("jobs", file_hash)
-        return self._execute_query(
+        cursor = self._execute_query(
             f"""\
         EXECUTE SERVICE
         COMPUTE_POOL =  {compute_pool}
         spec=@{stage}/{stage_dir}/{spec_filename};
         """
         )
+        return OutputData.from_cursor(cursor)
 
-    def desc(self, job_name: str):
-        return self._execute_query(f"desc service {job_name}")
+    def desc(self, job_name: str) -> OutputData:
+        cursor = self._execute_query(f"desc service {job_name}")
+        return OutputData.from_cursor(cursor)
```

Comment:
I'm not sure about it. I would return cursor here and do output formating in the command. To me this would be a better separation of responsibilities. WDYT?

Reply:
It is good idea when there is only one result. I see problem when you would like to return few cursors and strings. I added methods for specific types, so you have to return type of results to properly add them in command

Reply:
Or maybe it will not be that huge problem?

Reply:
I think that form test perspective and possible reusablity separating query executingand output constructing will be useful. 

To me this
```py
return OutputData.from_cursor(JobManager().desc(job_name))
```
is much more explicit and yields more information

Reply:
Moved it to commands